.github/workflows/phpunit-oci.yml

```diff
@@ -68,27 +68,28 @@ jobs:
       matrix:
         php-versions: ${{ fromJson(needs.matrix.outputs.php-version) }}
         server-versions: ${{ fromJson(needs.matrix.outputs.server-max) }}
-        oci-versions: ['11', '18', '21', '23']
```

Comment:
nop (talk is ahead here for testing, workflow will be upstreamed soon)

Reply:
reverted
`shivammathur/setup-php@...` was bumped to # v2.31.1, but could get it later from upstream